src/pages/Register/Register.js

```diff
@@ -1,5 +1,60 @@
+import React from 'react';
+import './Register.css';
+//import { useNavigate } from 'react-router-dom';
+import Header from '../../components/Header/Header';
+import Card from '../../components/Card/Card';
+import Button from '../../components/Button/Button';
+import Input from '../../components/Input/Input';
+import UploadButton from '../../components/UploadButton/UploadButton';
+
 function Register() {
+    return(
+    <>
+        <Header />
+            <div className='register-mainContainer'>
+                <div className='register-cardContainer'>
+                    <Card content={
+                        <div className='register-cardContent'>
+                             <div className='register-inputContainer'>
+                                <Input 
```

Comment:
you should wrap each of the inputs in a container with classNames so you can set the width and height of it